atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,6 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        # currently we only support v1+v2 schema1
```

Comment:
I think the confusion is that koji_import needs to know the list of all content_types available. And pulp_pull, at least for now, only needs to know whether or not v2 schema 2 is available.

Making pulp_pull perform the checks seems reasonable to me. We should that first, if that's the desired approach.

To query the registry, there's the helper method: util.get_manifest_digests
But that only queries for v2 (both schema 1 and schema 2).
Something similar must be added for v1 content, as I'm not aware of existing code that does this.